### ISSUE TITLE
PSX SCPH-1200 Analog Not Working

### ISSUE THREAD

**Issue by Vorenu5** [misteraddons/Reflex-Adapt#26]
The analog sticks on my SCPH-1200 PS1 controller are not registering correctly on PC or MiSTer. The analog LED on the controller is lit and the Reflex LCD displays dualshock mode for the controller.

I have tried modes from the following v1.05 firmwares:
Combo 12: PSX (My preferred combo)
Non-Nintendo: PSX+Gun
Combo 1: PSX+Gun+Jog

I've tested this same SCPH-1200 PS1 controller on a real PS1 and using a cheap PSX to USB adapter on PC and MiSTer and the analog sticks register as expected. I have also tested a SCPH-10010 PS2 controller and the analog sticks register correctly using the Reflex on PC. I'm currently not sure why the SCPH-1200 is not functioning correctly. Is this a possible firmware issue? I'm open to more testing if needed.

**Comment by misteraddons** [comment]
I've seen one other report of this exact controller not behaving. I got one to test and mine works fine. Trying to think of what might be different about these.

**Comment by Vorenu5** [comment]
I've tested the controller a bit further on PC and discovered if I hold one stick in a specific direction and move the opposite stick, some ranges of the stick will be detected.

For example: Holding right stick at the top-left diagonal will allow input ranges on the left stick to be detected when moved. Or if holding left stick at top-left diagonal, it allows input ranges on the right stick to be detected when moved.

**Comment by mattd121** [comment]
Can confirm this. Mine is the translucent blue SCPH-1200. Analog doesn't work, button is pushed, red light is lit. On a separate note I also tried a PS2 controller (black) and the analogs don't work with it either. I can try and test some grey ones I have stored away.

Hopefully a fix to the firmware can be made. I can send in a controller too if needed for testing.

**Comment by mattd121** [comment]
Found my other grey 1200. Analogs don't work.

Edit: After some googling, looks like the analog sticks have to be manually enabled in-game in MiSTer by the following combo: L3+R3+up

Works like a charm now!

**Comment by misteraddons** [comment]
Yeah, this is sadly a known issue. It seems to do with controller
initialization. On Discord, Kavika shared this:

“I got the PS One controller to work correctly with the v2.0.1 firmware by
plugging in a PS2 Dualshock first (which works perfectly), then pulling the
PS2 controller out of the snac cable and plugging in the PS One controller.
Then the PS One controller works perfectly.

This is true on both my Reflex Adapts, so it isn't something specific to
the hardware revision or a defective reflex adapt.”

Can any of you reproduce this behavior?

**Comment by misteraddons** [comment]
I made a dedicated SCPH-1200 test prerelease firmware here:
https://github.com/misteraddons/Reflex-Adapt/releases/tag/v2.01-psx1200-test-pre1

Please test **MPGPSX1200Test.hex** from that release and report back.

This test build adds explicit analog init + short analog re-init attempts for controllers that initially report digital, and uses a slightly slower PSX poll interval to probe timing sensitivity.
